proto/inbound.proto

```diff
@@ -233,3 +240,27 @@ message GrpcRoute {
     }
   }
 }
+
+message HttpLocalRateLimit {
+  // Overall rate-limit, which all traffic coming to this server should abide
+  // to. If unset no overall limit is applied.
+  Limit total = 1;
+
+  // Fairness for individual identities; each separate source, grouped by
+  // identity, will have this rate-limit.
+  Limit identity = 2;
+
+  // Limit overrides for traffic from a specific sources.
+  repeated Override overrides = 3;
+
+  // TargetRefs for traffic sources. Only ServiceAccounts supported for now.
+  repeated io.linkerd.proxy.meta.Metadata targets = 4;
```

Comment:
Proxies shouldn't know anything about service accounts or how to translate between kubernetes resources and identity strings. This all belongs exclusively in the control plane.

We also should probably be careful about calling this 'targets' when it is... sources?

Doesn't this belong on an override? Or how does this work if it is specified at the whole policy level?

This is probably where we want to look at some of the [authorization prior art](https://github.com/linkerd/linkerd2-proxy-api/pull/388/files#diff-d0176fc930f8bffcb11840a4073fb43374b3089a2a2cb2543a8250670a404c96R168-R170) to support identity matching while preserving future variants.

Reply:
Great points.
I've renamed that part to `clients`. And it does belong under overrides indeed, my mistake. 
And you're right about service accounts, I've refactored it a bit to use identities instead.